packages/expo/cli/utils/obj.ts

```diff
@@ -0,0 +1,46 @@
+/** `lodash.get` */
+export function get(obj: any, key: string): any | null {
+  const branches = key.split('.');
+  let current: any = obj;
+  let branch: string;
+  while ((branch = branches.shift())) {
+    if (!(branch in current)) {
+      return null;
+    }
+    current = current[branch];
+  }
+  return current;
+}
```

Comment:
Not blocking for this PR but should be able to replace this with optional syntax now

Reply:
well the usage is given a string of paths `['ios.splash.image']` access members.